src/wms/src/main/java/org/geoserver/wms/MapLayerInfo.java

```diff
@@ -337,6 +355,43 @@ public int hashCode() {
         return result;
     }
 
+    private boolean userMapCRSForWFSNG(
```

Comment:
the method that during a GetMap lookups the requested CRS inside FeatureTypeInfo metadata to skip re-projection. Method also verified that the datastore is WFS-NG configured withusedefaultsrs set to false.